scripts/draw_container_calculate/draw_container_calculate.gml

```diff
@@ -2,6 +2,9 @@
 /// @param data
 function draw_container_calculate(data)
 {	
+	//ensure the data is complete
+	update_data_with_components(data);
```

Comment:
I think with the simpler direction of using just `*`, we don't need this here now.

Reply:
It didn't need to be there in the first place, it's a duplicate. Good catch.
However, it is required to remove get plaintext without asterisks, otherwise the width and height will be wrong. Sure, we could use `string_replace_all`, but this should be more performant (it takes advantage of the string splitting we've already done)

Reply:
I'm confused, it seems to be removed now?

Reply:
There were two called to update_data_with_components. This was a duplicate. Now, it is (correctly) only called once.